<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataProcessClass</class>
 <widget class="QMainWindow" name="DataProcessClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataProcess</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>选择根目录：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_calGrad">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>calGrad</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_root">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_size">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_res">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>....</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_res">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>结果保存到：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_root">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>分组张数：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_calDis">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>calDis</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_data1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>data1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_data2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>data2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_calGrad_2D">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>2DGrad</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_calDis_batch">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>batchDis</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="dataprocess.qrc"/>
 </resources>
 <connections/>
</ui>
